src/components/icons/FacebookIcon.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { siFacebook } from 'simple-icons';
+
+interface FacebookIconProps {
+  className?: string;
+  size?: number;
+}
+
+const FacebookIcon: React.FC<FacebookIconProps> = ({
+  className = '',
+  size = 20,
+}) => {
+  return (
+    <svg
+      className={className}
+      width={size}
+      height={size}
+      viewBox='0 0 24 24'
+      fill='currentColor'
+      xmlns='http://www.w3.org/2000/svg'
+    >
+      <path d={siFacebook.path} />
+    </svg>
+  );
+};
+
+export default FacebookIcon;
```

Comment:
+5 other sections

no need to reinvent the wheel, we have [react-simple-icons](https://github.com/icons-pack/react-simple-icons)  as a maintained package from [simple-icons](https://www.npmjs.com/package/simple-icons)

see: https://github.com/icons-pack/react-simple-icons

this implementation can be condensed to 
```ts
import { SiReact } from '@icons-pack/react-simple-icons';

function BasicExample() {
  return <SiReact color='#61DAFB' size={24} />;
}
```